src/connect.rs

```diff
@@ -1005,7 +1005,7 @@ mod verbose {
                 } else if c == b'\0' {
                     write!(f, "\\0")?;
                 // ASCII printable
-                } else if c >= 0x20 && c < 0x7f {
+                } else if (0x20..0x7f).contains(&c) {
```

Comment:
I'm not sure this is clearer, really...

Reply:
@seanmonstar, Got your point, but as far as I know, as per the best practices we should incorporate range operator wherever we can, It was suggested but Clippy as well.
But if you want I can remove this change.
Thanks